docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2910,6 +2910,479 @@ have not been included. These are:
 #. We sample 100% of events. This is likely unrealistic, but the percent is probably very high still. 
 #. There are errors in SSN data, which are not modeled here. 
 
+Noise Functions
+^^^^^^^^^^^^^^^
+
+In order to have a realistic challenge with PRL, it is essential to have 
+noise added to the data. We currently divide noise into two types: 
+
+#. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
+#. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
+
+Column Noise
+''''''''''''
+
+To begin, we will start with defining column based noise. Some general rules 
+for all column based noise include: 
+
+- Rows eligible for errors is the probability of selecting a row to have a particular type of noise added. For example, 1% noise level for incorrect selection to type of tax form means 1% of rows will be selected to have the wrong value selected. This is somewhat more complicated for: OCR, phonetic, typographic, and numeric miswriting which is elaborated on below. 
+- Token noise level is a noise parameter that only applies to certian noise types and defines the amount of errors expected once a string is selected for noise. This parameter is also elaborated on below. 
+- The amount of each type of noise is individually configurable for each column in each observer. This means that the end user can select that: in the census sex has 2% incorrect selections. The minimum noise is 0% and the maximum if 5% for all column noise. For errors per row, the minimum is 0 and maximum is 5. 
```

Comment:
```suggestion
- The amount of each type of noise is individually configurable for each column in each observer. This means that the end user can specify, for example, that in the census, sex has 2% incorrect selections. The minimum noise is 0% and the maximum if 5% for all column noise. For errors per row, the minimum is 0 and maximum is 5. 
```
This was a bit confusing when I first read it because you say that noise is configurable, but then it sounds like you're specifying a definite value, so here's an attempt to make it more clear.

Also, what's the reason we're specifying a maximum noise level, and how did we choose it? It seems like there's no reason it would be easier to have a maximum than not to have a maximum, except for the sake of not having the code take forever to run if the noise level is very high.

Reply:
Thanks, I see your point on clarity! 

For the max, we can reassess this later. I just thought it might be helpful for the engineers to have a max value they should account for in designing things. But you're right, we could set the max to 100%.